gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/SearchEngineServiceTest.java

```diff
@@ -535,6 +537,25 @@ public void shouldFindWithoutEnvironmentId() {
         assertEquals(Arrays.asList("api-1", "api-2"), new ArrayList(matches.getDocuments()));
     }
 
+    @Test
+    public void shouldFindWithExcludedFilters() {
+        Map<String, Object> filters = new HashMap<>();
+        filters.put(FIELD_API_TYPE_VALUE, Arrays.asList("api-1", "api-2"));
+        Map<String, Collection<String>> excludedFilters = new HashMap<>();
+        excludedFilters.put(FIELD_DEFINITION_VERSION, Arrays.asList(DefinitionVersion.V1.getLabel(), DefinitionVersion.V4.getLabel()));
```

Comment:
maybe we should test with more than one exclusion filter. Maybe on the tag or the name ?